### PR TITLE
Add event materials

### PR DESCRIPTION
**Pull request by momack2**

### CODE REVIEW

**Comment by momack2** [comment]
Ref https://github.com/ProtoSchool/organizing/issues/63

File: README.md [hunk 1]
```diff
@@ -85,6 +86,14 @@ Find the list at:
...
 
+### IPFS Event Materials
+Want to host an event to share IPFS with others, or bring IPFS to another event? Feel free to use these materials to make your event(s) awesome!
+
+- **How IPFS Works Overview:** [@stebalien](http://github.com/@stebalien), go-ips Captain and Lead Maintainer, created a great presentation template for "[How IPFS Works (approximately)](https://docs.google.com/presentation/d/1wyrTNTERJV23R7PyLrNloBauygdq1SXerijvEUG62xI/edit)". _Feel free to reuse or fork this deck to help break down the inner magic of IPFS for others._ [Here's a recording](https://drive.google.com/file/d/1yRaBqjdu9mWeVzGl-pSPfwZ7XUjb2rwy/view) of this deck being used in practice to explain how IPFS works (from the October 2019 Osaka IPFS Meetup).
```
Comment (Stebalien) [inline README.md]:
```suggestion
- **How IPFS Works Overview:** [@stebalien](http://github.com/@stebalien), go-ipfs Captain and Lead Maintainer, created a great presentation template for "[How IPFS Works (approximately)](https://docs.google.com/presentation/d/1wyrTNTERJV23R7PyLrNloBauygdq1SXerijvEUG62xI/edit)". _Feel free to reuse or fork this deck to help break down the inner magic of IPFS for others._ [Here's a recording](https://drive.google.com/file/d/1yRaBqjdu9mWeVzGl-pSPfwZ7XUjb2rwy/view) of this deck being used in practice to explain how IPFS works (from the October 2019 Osaka IPFS Meetup).
```

go-ips -> go-ipfs

File: README.md [hunk 2]
```diff
@@ -59,17 +60,17 @@ IPFS is an open source project with a large and open community. Listing out all
...
 ## Contributing Guidelines
 
 - [Contributing Guidelines](./CONTRIBUTING.md)
 - [Contributing Guidelines for JavaScript Projects](./CONTRIBUTING_JS.md)
 - [Contributing Guidelines for Golang Projects](./CONTRIBUTING_GO.md)
 - [Code of Conduct](./code-of-conduct.md).
 
+## Swag
+
+We have stickers and we would love to send them to you. If you're hosting a series of IPFS meetups, we'd also love to send you T-shirts to distribute! Send your request directly to swag+requests@protocol.ai.
```
Comment (terichadbourne) [inline README.md]:
```suggestion
We have stickers and we would love to send them to you. If you're hosting a series of IPFS meetups, we'd also love to send you T-shirts to distribute! Send your request directly to [swag+requests@protocol.ai](mailto:swag+requests@protocol.ai).
```

File: README.md [hunk 3]
```diff
@@ -85,6 +86,14 @@ Find the list at:
...
+  - Still have questions? [IPFS Simply Explained](https://www.youtube.com/watch?v=5Uj6uR3fp-U) does a great job visualizing how IPFS addresses some key use cases and needs to upgrade the internet.
+- **IPFS Deep Dive Workshops:** These two videos - [How IPFS Deals with Files](https://www.youtube.com/watch?v=Z5zNPwMDYGg&list=PLuhRWgmPaHtSsHMhjeWpfOzr8tonPaePu&index=2&t=0s) and [The Lifecycle of Data on the Dweb](https://www.youtube.com/watch?v=fLUq0RkiTBA&list=PLuhRWgmPaHtSsHMhjeWpfOzr8tonPaePu&index=3&t=8s) - are great workshops on how core components of the IPFS network fit together. Feel free to watch these together, or just use as inspiration.
+- **Context and Vision for IPFS:** In [Why IPFS?](https://www.youtube.com/watch?v=zE_WSLbqqvo) and [Building Web3](https://www.youtube.com/watch?v=pJOG5Ql7ZD0) Juan Benet talks about how IPFS fits into a longer-scale evolution of technology and our progress on building the new set of capabilities to power web3. The amazing video in "Why IPFS" that sets the stage for our place in the cosmos is "[Our Story in 1 Minute](https://www.youtube.com/watch?v=ZSt9tm3RoUU)" by melodysheep.
```
Comment (terichadbourne) [inline README.md]:
```suggestion
- **Context and Vision for IPFS:** In [Why IPFS?](https://www.youtube.com/watch?v=zE_WSLbqqvo) and [Building Web3](https://www.youtube.com/watch?v=pJOG5Ql7ZD0), Juan Benet talks about how IPFS fits into a longer-scale evolution of technology and our progress on building the new set of capabilities to power web3. The amazing video in "Why IPFS" that sets the stage for our place in the cosmos is "[Our Story in 1 Minute](https://www.youtube.com/watch?v=ZSt9tm3RoUU)" by melodysheep.
```

**Comment by terichadbourne** [comment]
@momack2 Just checking to see if this is ready to merge or if you'd like to add anything about copyrights first.

File: README.md [hunk 4]
```diff
@@ -85,6 +86,14 @@ Find the list at:
 
 [ProtoSchool](https://proto.school) is a community-driven open source project dedicated to making it easy to get started with decentralized web technologies like IPFS through [interactive online tutorials](https://proto.school/#/tutorials) and community learning events led by local chapters around the world. To locate an event near you, check out our [chapter list](https://proto.school/#/chapters). Don't see a ProtoSchool chapter in your neighborhood? [Start your own!](https://proto.school/#/host)
 
+### IPFS Event Materials
+Want to host an event to share IPFS with others, or bring IPFS to another event? Feel free to use these materials to make your event(s) awesome!
```
Comment (momack2) [inline README.md]:
```suggestion
Want to host an event to share IPFS with others, or bring IPFS to another event? Feel free to use these materials to make your event(s) awesome! Everything here is [Creative Commons 4.0](https://creativecommons.org/licenses/by-sa/4.0/) - so feel free to adapt or remix!
```

**Comment by momack2** [comment]
I think it's good to go - added a quick note. =]

**Pull request merged**